persist/sqldb/sqldb.go

```diff
@@ -53,9 +54,44 @@ func CreatePostGresDBSession(kubectlConfig kubernetes.Interface, namespace strin
 	}
 
 	if cfg.SSL {
-		if cfg.SSLMode != "" {
+		if cfg.SSLMode != "" && cfg.SSLMode != "disable" {
+			pgCertPath := "./pgcerts"
```

Comment:
can we use the absolute path here and get rid of `WORKDIR /home/argo` from Dockerfile?

Reply:
@rohankmr414  we ca use absolute path. I thought argo server already using workdir in Dockerfile so I thought same thing we can use for argo controller.

Reply:
I don't see any issues with it as well, it's just an extra change that can be avoided.

cc: @sarabala1979

Reply:
Can we add another element in structs to configure the path and have a default value too?

Reply:
@sarabala1979  @rohankmr414  then we need to use constructor for this field in the struct

Reply:
@reddymh yes you can add a field to the spec

Reply:
@rohankmr414 @sarabala1979  added field to the spec and default vault if the variable is not configured in configmap or not defined.
Please review and approve the same